frontends/bmd/src/pages/card_error_screen.tsx

```diff
@@ -21,6 +27,11 @@ export function CardErrorScreen({
     <Screen white>
       <Main>
         <MainChild center>
+          <RotateCardImage
+            aria-hidden
+            src="/images/rotate-card.svg"
+            alt="rotate-card"
+          />
```

Comment:
Nit: I'd like to do this with less code because this is going to become a pattern… but it can be optimized later.

Preferably, move the image (content) into the `Prose` block because it is part of the "prose". Wrap it in a `<p>` tag for default "prose" margin spacing. 

But to do the centering, I guess you'd need some CSS as there isn't anything in the Prose component for centering images yet! So the centering could be inline on the image: `style={{ width: '300px', margin: '0 auto' }}`

Reply:
I tried this but it put the image too high up due to the h1 top margin. Though the centering did work automatically. So gonna leave as is this time around